### PR TITLE
EPP-52 Setting up dev environment

### PR DESCRIPTION
**Pull request by MikeMaratheftis**
## What? 
[EPP-52](https://collaboration.homeoffice.gov.uk/jira/browse/EPP-52) - Create dev environment 

## Why? 
Allow for the development of EPP 

## How? 
Using HOF skeleton, renaming the placeholder names and IDs according to the EPP designs. 

## Testing?
Tested on local machine 

## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "hof-epp-form",
```
Comment (PaolaDMadd-Pro) [inline package.json:2]: Please verify if the name of the app needs to reflect the name of the git repository or hosting platform name

File: package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "version": "1.0.0",
+  "description": "Simple standard form made using HOF",
```
Comment (PaolaDMadd-Pro) [inline package.json:4]:
```suggestion
  "description": "Explosive precursors and poisons within top-level forms",
```

**Review by PaolaDMadd-Pro — CHANGES_REQUESTED** [review]
Good work, I added small suggestions, that won't preclude the merging.
@william-gu-hof

File: config.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+'use strict';
+/* eslint no-process-env: 0 */
+
+const env = process.env.NODE_ENV || 'production';
+
+module.exports = {
+  PRETTY_DATE_FORMAT: 'Do MMMM YYYY',
```
[the remark this answers is not in the document]
Comment (MikeMaratheftis) [reply config.js:7]: The correct format is 'DD MMMM YYYY' 

Good spot :)

**Review by PaolaDMadd-Pro — APPROVED** [review]
looks good to me

**Pull request merged**